src/shims/manual/mscorlib.csproj

```diff
@@ -4,36 +4,83 @@
-    <SeedTypePreference Include="System.Reflection.Emit.GenericTypeParameterBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.ILGenerator" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.Label" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.LocalBuilder" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.MethodBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.ModuleBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.ParameterBuilder" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.PropertyBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Reflection.Emit.SignatureHelper" Assembly="System.Reflection.Emit.ILGeneration" />
-    <SeedTypePreference Include="System.Reflection.Emit.TypeBuilder" Assembly="System.Reflection.Emit" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.DefaultInterfaceAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.EventRegistrationToken" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.EventRegistrationTokenTable`1" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.IActivationFactory" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.InterfaceImplementedInVersionAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArrayAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.ReturnValueNameAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.WindowsRuntimeMarshal" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
-    <SeedTypePreference Include="System.Runtime.InteropServices.WindowsRuntime.WriteOnlyArrayAttribute" Assembly="System.Runtime.InteropServices.WindowsRuntime" />
+    <Compile Include="mscorlib.forwards.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsAot)' == 'true' or '$(TargetsUap)' == 'true'">
+    <ReferencePath Include="$(_RuntimePath)System.Reflection.Emit.dll" Aliases="System_Reflection_Emit" />
```

Comment:
Just curious, why the underscores?

Reply:
Maybe this should set RefPath instead. https://github.com/dotnet/corefx/blob/master/src/shims/manual/Directory.Build.props